test/api/v3/unit/models/group.test.js

```diff
@@ -460,73 +457,67 @@ describe('Group Model', () => {
         };
       });
 
-      it('throws an error if no uuids or emails are passed in', (done) => {
-        try {
-          Group.validateInvitations(null, null, res);
-        } catch (err) {
-          expect(err).to.be.an.instanceof(BadRequest);
-          expect(res.t).to.be.calledOnce;
-          expect(res.t).to.be.calledWith('canOnlyInviteEmailUuid');
-          done();
-        }
+      it('throws an error if no uuids or emails are passed in', async () => {
+        await expect(Group.validateInvitations(null, null, res)).to.eventually.be.rejected.and.eql({
+          httpCode: 400,
+          message: 'Bad request.',
```

Comment:
The message returned in the call to ``Group.validateInvitations`` is ``Bad request.``.

The only expectation about the call of this function is that it will fail with Bad Request. The reason of fail (``canOnlyInviteEmailUuid``) is on the ``res`` variable.

So, I can't see a way to do what you're requesting. Sorry.